docs/release-notes/artifacts/pr0175.yaml

```
@@ -0,0 +1,15 @@
+# --- Release notes artifact ----
+
+schema_version: 1
+changes:
+  - title: Feat: Add http-server-close support

```

```suggestion
  - title: Added http-server-close support

```
I think we should probably remove or recommend against using "conventional commit" style language in the change artifact titles. WDYT?